client/src/components/Workflow/Editor/Node.vue

```diff
@@ -156,14 +156,14 @@ export default {
         const step = this.step;
         const el = this.$el;
         if (step.position) {
-            el.style.top = step.position.top + "px";
-            el.style.left = step.position.left + "px";
+            el.style.top = (step.position.top + this.offsetToPreventNodeOverlap(100)) + "px";
```

Comment:
First of all, thank you so much for working on this. I noticed that if you change the position here the positions of the nodes are always reshuffled upon loading of a saved workflow, not only when they are inserted. You probably only need the offset randomization below. Also please use shorter branch names, and shorter function names in alignment to the function names used in a particular file for consistency. In this particular case I suggest to just declare the constant (e.g. const OFFSET_RANGE = 100) at the top of the component and directly calculate as you do without an additional function but with a local variable e.g. `const offset =  Math.floor(Math.random()*OFFSET_RANGE)`.

Reply:
@guerler Thank you for the helpful feedback and thanks for being my first reviewer for the galaxy repo :) I am new to using the tool and what you say about not needing to apply the random offset value for "existing" nodes (eg. in saved Workflows), makes perfect sense. I have added a new commit to account for the issue. Moving forward, I hope I have the opportunity to learn more about the tool and its many use cases. Thanks again.